packages/sass-render/bin/sass-render.js

```diff
@@ -51,9 +52,28 @@ if (!source) {
 
 glob(source, (err, files) => {
   files
-    .filter(file => !path.basename(file).startsWith('_'))
-    .forEach(file => {
-      sassRender(file).catch(error => {
+    // .filter(file => !path.basename(file).startsWith('_'))
+    .map((file) => {
+      // if not partial, process like regular file
+      const isPartial = path.basename(file).startsWith('_');
+      if (!isPartial) {
+        return file;
+      }
+
+      // if this is a partial, try to find its parent searching for `${parent_dir}.scss`
+      const mainScss = `${path.dirname(file)}.scss`;
```

Comment:
Maybe `parentScss`, per comment?

But this logic seems overly simplistic - for example, what parent does this logic determine for https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/scss/cxl-app-layout/_layout.scss

Feels like this has to be something that understands SASS.

I've noticed dart-sass core built-in watch function is able to compile import parents automatically.. But I'm not sure how to make that work here.

Your thoughts?

Reply:
My question would be what is the reason to develop `watcher.js` and `sass-render` package as custom modules, why not to use existing `Webpack` plugins for SASS -> CSS compilation?
I've always used libraries that can handle this perfectly and are very easy to use.
Example: https://symfony.com/doc/current/frontend/encore/simple-example.html#configuring-encore-webpack

Reply:
`I've noticed dart-sass core built-in watch function is able to compile import parents automatically.` what function do you have in mind?

Reply:
> My question would be what is the reason to develop watcher.js and sass-render package as custom modules, why not to use existing Webpack plugins for SASS -> CSS compilation?

It was shipped such from upstream when we started, because the build process was Pika / Rollup-based.

Probably to avoid webpack's inherent complexity.

But today, we are currently working on https://github.com/conversionxl/aybolit/pull/66 which brings Webpack into our build process. Maybe this is the a suitable target to also refactor SCSS build with? Are you able to build a webpack-based PR on top of #66 branch?